src/Compilers/VisualBasic/Portable/Symbols/SynthesizedSymbols/SynthesizedParameterSymbol.vb

```diff
@@ -233,6 +233,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return _name
             End Get
         End Property
+
+        Friend NotOverridable Overrides Sub AddSynthesizedAttributes(compilationState As ModuleCompilationState, ByRef attributes As ArrayBuilder(Of SynthesizedAttributeData))
+            Dim compilation = Me.DeclaringCompilation
+            If Type.ContainsTupleNames() AndAlso
+                compilation.HasTupleNamesAttributes AndAlso
```

Comment:
Update: I disabled the two checks (HasTupleNames && CanEmitSpecialType(String)) (by throwing an exception if either was false) for both VB and C# and re-ran tests, and all tests passed. Should definitely check in with someone on what the thinking there was - it looks like git blame is saying Andy added those blocks.